rskj-core/src/main/java/org/ethereum/core/genesis/GenesisLoaderImpl.java

```diff
@@ -54,6 +56,13 @@
  * 4. Registers the genesis state root in the state root handler
  */
 public class GenesisLoaderImpl implements GenesisLoader {
+
+    static class GenesisLoaderExceptopn extends RuntimeException {
```

Comment:
Response to sonar cloud 
![Screenshot 2024-04-02 at 12 19 57](https://github.com/rsksmart/rskj/assets/36888576/e7579dc9-4bfb-40a8-9793-610d0e0408c9)